src/adv.py

```diff
@@ -49,3 +53,56 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+player = Player(room['outside'])
+user_input = None
+
+# Clear the terminal
+os.system('cls' if os.name == 'nt' else 'clear')
+
+# Print the title
+print("\n\n  Welcome to \n    THE GAME")
+
+# Begin REPL loop
+while user_input != "q":
+
+    # if this is not the first loop, determined by the user_input clear the console
+    if(user_input != None):
```

Comment:
use of logical operators in Python is not very "pythonic" and might throw an error in some cases. Use regular old English instead.